astroquery/casda/tests/test_casda.py

```diff
@@ -117,6 +125,41 @@ def isclose(value1, value2, abs_tol=1e-09):
     return abs(value1 - value2) < abs_tol
 
 
+def fake_login(casda, username, password):
+    casda.USERNAME = username
+    casda._auth = (username, password)
+    casda._authenticated = True
+
+
+def test_login_no_default_user():
+    casda = Casda()
+    assert casda._authenticated is False
+    assert casda.USERNAME == ''
+
+    with pytest.raises(LoginError) as excinfo:
+        Casda.login()
+
+    assert "If you do not pass a username to login()," in str(excinfo.value)
```

Comment:
```suggestion
    with pytest.raises(LoginError, match=r"If you do not pass a username to login\(\),") as excinfo:
        Casda.login()
```

Reply:
I've made this change made all exception tests